src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -226,7 +238,29 @@ def _search(
                 query_kwargs.update(extra_query_kwargs)
 
             if query_kwargs["sort_by"] == "betterPriority":
-                query_kwargs["aggregate_kwargs"] = self.build_better_priority_sort_kwargs(request)
+                choice = None
+                if features.has(
+                    "organizations:better-priority-sort-experiment",
+                    organization,
+                    actor=request.user,
+                ):
+                    choice = expt_manager.get(
+                        "PrioritySortExperiment", org=organization, actor=request.user
+                    )
+                # force into variant1 for internal testing
+                if features.has(
+                    "organizations:issue-list-better-priority-sort",
+                    organization,
+                    actor=request.user,
+                ):
+                    choice = "variant1"
+
+                if choice == "baseline":
```

Comment:
@ceorourke I think this needs to be an elif so we don't wipe out the variant1 override

Reply:
What do you mean? It's just checking if the choice is baseline and setting the sort to last seen here. It's not overwriting the choice value.

Reply:
@ceorourke but its going to make `query_kwargs["sort_by"] = "date"` which isn't what we want if the feature flag is enabled, right?

Reply:
If they're in the experiment they could fall into baseline, variant1, or variant2. All I'm doing here is setting baseline to last seen.

Reply:
@ceorourke right...but if they have the `organizations:issue-list-better-priority-sort` set to true then we want to force to variant1 and ignore the experiment value.

Reply:
How does it not do that?

Reply:
@ceorourke you're right and I'm bad at reading code